JustSaying/JustSayingFluently.cs

```diff
@@ -184,14 +185,21 @@ public virtual void Publish(Message message)
         /// Publish a message to the stack, asynchronously.
         /// </summary>
         /// <param name="message"></param>
-        public virtual async Task PublishAsync(Message message)
+        public virtual async Task PublishAsync(Message message) => await PublishAsync(message, CancellationToken.None);

```

Comment:
Don't strictly need the `async` or `await` here as no other work is done.

Reply:
Good point